tests/analyze_ecl_failure.sh

```diff
@@ -24,7 +24,7 @@ do
     for line in $alines
     do
       abs_new=`echo $line| awk -F ' ' '{print $5}'`
-      abs_new=`echo ${abs_new: : -1} | awk '{printf sprintf("%.16f", $1); }'`
+      abs_new=`echo ${abs_new} | sed -e 's/\.$//g' | awk '{printf sprintf("%.16f", $1); }'`
```

Comment:
The fix was proposed by @akva2, so I'll have to defer to him on this. I think the idea is to strip the dot because awk does not understand floats with dots but no digits after the dot.